api/server/api_0.cc

```diff
@@ -474,10 +474,8 @@ void Api0::AppConfig(const MessageV0_Request &req, MessageV0_Response *res) {
     app::log.Info("Application configuration");
     auto config = req.app_config();
     std::string err;
-    if (config.has_log_level()) {
-        if (!app::log.SetLogLevel(config.log_level()))
-            err = err + "log level: failed ";
-    }
+    if (config.has_log_level() && (!app::log.SetLogLevel(config.log_level())))
```

Comment:
` if (config.has_log_level() && !app::log.SetLogLevel(config.log_level()))`
there is an useless level of parentheses around app::log.SetLogLevel